ciao_contrib/_tools/specextract.py

```diff
@@ -443,7 +443,7 @@ def _check_event_stats(self,file,refcoord_check=None,
         if counts == "0":
             try:
                 if refcoord_check.lower() in ["","none","indef"]:
-                    raise IOError(f"{file} has zero counts. Check that the region format is in sky pixels coordinates.")
+                    raise IOError(f"{file} has zero counts. Check that the region correct (e.g. wrong region, coordinates not in sky pixels or degrees).")
```

Comment:
Do we want to put the workaround to run this anyway in the message or do we thing it's better to leave that for expert users to figure out themselves on the thought that in most cases zero counts actually indicates an error in the region?

```suggestion
                    raise IOError(f"{file} has zero counts. Check that the region correct (e.g. wrong region, coordinates not in sky pixels or degrees) or use parameter "refcoord"")
```

Reply:
Let's be explicit...

Reply:
Redid the suggestion to fix a typo (use of `"` rather than `'`)